src/config.py

```diff
@@ -1,4 +1,12 @@
 from os import getenv
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+client = os.getenv("CLIENT_ID")
```

Comment:
Let us plug these in the `APP_CONFIG` dict below. Then we can access it via e.g. `request.app['KEY']`